server/routes/dashBoard.js

```diff
@@ -0,0 +1,250 @@
+const express = require("express");
+const router = express.Router();
+const { updateData } = require("../util/util");
+const checkToken = require("../auth/validateToken");
+
+// Models;
+const { Task, Column, Dashboard } = require("../models/Dashboard");
+
+//@CreateBoard
+router.get("/dashboard", checkToken, async (req, res) => {
+  const userId = req.decoded.id;
+
+  try {
+    let result = await Dashboard.findOne({ user: userId });
+    res.status(200).send(result);
+  } catch (err) {
+    res.status(404).json({ error: "Dashboard does not exist" });
+  }
+});
+
+//Add Dashboard @Done
+router.post("/dashboard", async (req, res) => {
+  const { dashboardTitle, token } = req.body;
+
```

Comment:
checkToken function should also be added here

Reply:
set checkToken to all path